Resources/public/script/httplug.js

```diff
@@ -0,0 +1,132 @@
+
+
+/**
+ * Toggle hide/show on the message body
+ */
+function httplug_toggleBody(el) {
+  var bodies = document.querySelectorAll(".httplug-http-body");
```

Comment:
Well, doesn't really matter after all. But maybe we should add a rule to the editorconfig, otherwise some IDEs might want to indent differently.